services/movie_session.py

```diff
@@ -0,0 +1,43 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    movie = Movie.objects.get(id=movie_id)
+    hall = CinemaHall.objects.get(id=cinema_hall_id)
+    MovieSession.objects.create(show_time=movie_show_time,
```

Comment:
Or you could do it in one line passing ids on movie session create `movie_id=movie_id`

Reply:
still not fixed @Alexandr-Politov